zk/types/zk_types_test.go

```diff
@@ -50,3 +52,35 @@ func Test_L1InjectedBatchMarshallUnmarshall(t *testing.T) {
 
 	require.Equal(t, input, result)
 }
+
+func Test_L1InjectedBatch_UnmarshalJSON(t *testing.T) {
```

Comment:
Thanks for adding this test. If you could add a test for empty file that would be great.

Reply:
Fixed https://github.com/0xPolygonHermez/cdk-erigon/pull/1312/commits/13865ecf710b2539273a9356c051c6e27d7cf77f